src/MongoDB/BulkWrite.c

```diff
@@ -188,7 +188,7 @@ static PHP_METHOD(BulkWrite, __construct)
 		ordered = php_array_fetchc_bool(options, "ordered");
 	}
 
-	intern->bulk = phongo_bulkwrite_init(ordered);
+	intern->bulk = mongoc_bulk_operation_new(ordered);
```

Comment:
This is really unrelated to the patch, you just removed ``phongo_bulkwrite_init`` as it only did this one thing. I would suggest you don't do this in this single commit.